### DISCUSSION TITLE
[Capitolo 1] Esercizio 1.2.2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#163] [Esercizi - Viaggi/Piazza]
**Esercizio 1.2.2:** Si provi che le condizioni che definiscono una relazione di equivalenza sono indipendenti, dando:
**a)** un esempio di relazione riflessiva e simmetrica, ma non transitiva;
**b)** un esempio di relazione riflessiva e transitiva, ma non simmetrica;
**c)** un esempio di relazione simmetrica e transitiva, ma non riflessiva.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/84519330-7f30-4eb4-9dd5-913ab88e3499)
